packages/storybook-builder-vite/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "storybook-builder-vite",
-    "version": "0.0.12",
+    "version": "0.0.13",
```

Comment:
Can you please remove the change to the version number?  We will do that separately when we are ready to release a new version.

Reply:
I have put them back to the 0.0.12 versions